content/webapp/services/prismic/transformers/body.ts

```diff
@@ -492,13 +494,6 @@ export function transformBody(body: Body): BodySlice[] {
         case 'searchResults':
           return transformSearchResultsSlice(slice);
 
-        // Quote used to be the one we offered - it still exists in 189 instances
-        // We now offer QuoteV2 instead, which has the same fields.
-        // They both use the same renderer - we just need to still support legacy Quote
-        // TODO: One day, manually move them all over to use QuoteV2 and clear out any
-        //       old reference to Quote. Ideally rename QuoteV2 to Quote.
-        //       The best way to do this is up for debate:
-        //       https://github.com/wellcomecollection/wellcomecollection.org/pull/9979#issuecomment-1602448601
         case 'quote':
         case 'quoteV2':
```

Comment:
Will we get to remove this once the script has been run and we've moved over completely? :')